doc/data-mining-library/source/reference/data.io.rst

```diff
@@ -56,7 +53,7 @@ A three-line header consists of:
    * ``weight`` (or ``w``) — the feature marks the weight of examples (in
      algorithms that support weighted examples),
    * ``ignore`` (or ``i``) — feature will not be imported,
-   * ``<key>=<value>`` custom attributes.
+   * ``<key>=<value>`` custom attributes. Such an example is ``type=image``, which denotes the path to the image (locally or online) or ``include=True``, which defines the variables as a text variable.
```

Comment:
Maybe add some context here. `include=True` does not have any general meaning. E.g.


``<key>=<value>`` custom attributes recognized in specific contexts, for instance `color`, which defines the color palette when the variable is visualized, or `type=image` which signals that the variable contains a path to an image.

Reply:
Perhaps we need to change the `include=True` into `text_feature=True` or something more descriptive. As far as I know, include is used exclusively for denoting text features.

Reply:
@PrimozGodec What do you think?

Reply:
I have kept @janezd 's proposal. Include should be one day renamed to a nicer pair, i.e. `type=text`.